app/models/slack_announement.rb

```diff
@@ -0,0 +1,20 @@
+require 'httparty'
+SLACK_TOKEN = ENV['SLACK_TOKEN']
+CHANNEL = "CEVDW26C8".freeze
+SLACK_METHOD = "https://slack.com/api/channels.history".freeze
+COUNT = 20.freeze
+
+class Slack
+  def initialize
+    if !response || response["ok"] == false ?
```

Comment:
```suggestion
    if !response || response["ok"] == false
```

Should fix the syntax error